docker/Dockerfile-qa

```diff
@@ -0,0 +1,20 @@
+FROM jakzal/phpqa:alpine
+
+RUN set -xe \
+    && apk add --update openssl ca-certificates \
+    && apk add --no-cache --virtual .php-deps \
+        libpq \
+        icu-libs \
+        git \
+    && apk add --no-cache --virtual .build-deps \
+        $PHPIZE_DEPS \
+        postgresql-dev \
+        icu-dev  \
+    && docker-php-ext-install \
+        pdo_pgsql \
+        pgsql \
+        bcmath \
+        intl \
```

Comment:
xdebug?

Reply:
It depends, I'm not sure this is required for the qa tools 🤔

Reply:
TIL php-dbg can collect coverage data. 👍

Reply:
BTW @jakzal confirmed in https://github.com/jakzal/phpqa/issues/18#issuecomment-345235812 you can already use phpdbg from his image so this can be ignored.